public/api_docs/locales/locale.js

```diff
@@ -0,0 +1,47 @@
+define([
+    './locales/ca.js',
+    './locales/de.js',
+    './locales/es.js',
+    './locales/fr.js',
+    './locales/it.js',
+    './locales/nl.js',
+    './locales/pl.js',
+    './locales/pt_br.js',
+    './locales/ru.js',
+    './locales/zh.js',
+    './locales/zh_cn.js'
+], function() {
+    var langId = (navigator.language || navigator.userLanguage).toLowerCase().replace('-', '_');
```

Comment:
please skip everything inside `public/api_docs`, as they are autogenerated. However to answer your question, I think `navigator` is within the `window` object in JS